obspy/taup/tests/test_tau.py

```diff
@@ -107,6 +112,64 @@ def test_p_iasp91_manual(self):
         self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
         self.assertEqual(p_arrival.purist_name, "P")
 
+    @unittest.skipIf(not geobase.HAS_GEOGRAPHICLIB,
+                     'Module geographiclib is not installed')
+    def test_p_iasp91_geo_manual(self):
+        """
+        Manual test for P phase in IASP91 given geographical input.
+
+        This version of the test is used when geographiclib is installed
+        """
+        m = TauPyModel(model="iasp91")
+        arrivals = m.get_travel_times_geo(source_depth_in_km=10.0,
+                                          source_latitude_in_deg=20.0,
+                                          source_longitude_in_deg=33.0,
+                                          station_latitude_in_deg=55.0,
+                                          station_longitude_in_deg=33.0,
+                                          phase_list=["P"])
+        self.assertEqual(len(arrivals), 1)
+        p_arrival = arrivals[0]
+
+        self.assertEqual(p_arrival.name, "P")
+        self.assertAlmostEqual(p_arrival.time, 412.43, 2)
+        self.assertAlmostEqual(p_arrival.ray_param_sec_degree, 8.612, 3)
+        self.assertAlmostEqual(p_arrival.takeoff_angle, 26.74, 2)
+        self.assertAlmostEqual(p_arrival.incident_angle, 26.69, 2)
+        self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
+        self.assertEqual(p_arrival.purist_name, "P")
+
+    def test_p_iasp91_geo_fallback_manual(self):
+        """
+        Manual test for P phase in IASP91 given geographical input.
+
+        This version of the test checks that things still work when
+        geographiclib is not installed.
+        """
+        has_geographiclib_real = geobase.HAS_GEOGRAPHICLIB
+        geobase.HAS_GEOGRAPHICLIB = False
+        taup_geo.HAS_GEOGRAPHICLIB = False
+        tau.HAS_GEOGRAPHICLIB = False
```

Comment:
I don't think you avoid this by testing calc_dist rather than testing get_travel_times_geo. You still need to convince taup.taup_geo and geodetics.base to exercise the "with geographiclib" and "without geographiclib" code paths which, if we don't change the way `HAS_GEOGRAPHICLIB` is imported, means we need to set at least two variables. And I think  you *really* want to know if calc_dist() fails to return something sane if geographiclib is not installed, even if you happen to be testing on a machine where it is installed.

One option would be to call calc_vincenty_inverse() directly from the fallback code in taup.taup_geo (so we avoid having decide if geographiclib is installed in gps2dist_azimuth) but then we end up duplicating code and we would still need to change the `if HAS_GEOGRAPHICLIB:` to a `try ... except` block.

Maybe what we should do is change `from ..geodetics.base import HAS_GEOGRAPHICLIB` to `import geodectics.base` and `HAS_GEOGRAPHICLIB` to `geodectics.base.HAS_GEOGRAPHICLIB` in taup.tau and taup.taup_geo so that we only need to change `geodectics.base.HAS_GEOGRAPHICLIB` in the tests?